### DISCUSSION TITLE
My Student ID is digital which makes taking a photo with my laptop difficult.

### DISCUSSION THREAD

**Discussion by nathansantiago** [community/community#131466] [GitHub Education]
### Hi everyone,

I am trying to apply for github education benefits, however my school issued ID is digital and I am struggling to get a photo with my laptop's webcam.

**Comment by PipoAT** [comment]
@nathansantiago If you are not able to submit your school issued ID, you can submit other documentation that proves enrollment such as transcripts, class schedule, etc.

This [discussion link](https://github.com/orgs/community/discussions/111352) provides an overview of the application process for student benefits and lists out acceptable documents as alternatives to your ID.

I hope this helps!
